db/schema.rb

```diff
@@ -36,4 +36,15 @@
   add_index "users", ["email"], name: "index_users_on_email", unique: true
   add_index "users", ["reset_password_token"], name: "index_users_on_reset_password_token", unique: true
 
+  create_table "wikis", force: :cascade do |t|
+    t.string   "title"
```

Comment:
Are you validating the data in the columns? There's two approaches to this, and I actually suggest doing both:

1) Add DB constraints. Notice the `null: false` on lines 44 and 45. This constraint ensures that null values cannot be inserted into those columns. For boolean columns, consider `null: false, default: false` -- this avoids creating a three state boolean where potential values are true, false and nil.

2) Validate in the model. `validates :title, :body, :user_id, presence: true`. It's nice to add these to the model so that when a validation fails, you get a nice error message from Rails.